AstroAdapt.GraphQL/ModuleInfo.cs

```diff
@@ -0,0 +1 @@
+[assembly: Module("AstroTypes")]
```

Comment:
You can register per assembly a module name. The module name is the name of the generated type module. The source generator will create an extension method `AddAstroTypes` in our case that can be chained in the configuration located in the `Program.cs`.